### ISSUE TITLE
[ISSUE]: Manually Setting Bump Type with /overrideconfig increment=X does not affect version

### ISSUE THREAD

**Issue by dbieber-rfideas** [GitTools/GitVersion#4117]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched [issues](https://github.com/GitTools/GitVersion/issues) to ensure it has not already been reported

### GitVersion package

GitVersion.Tool

### GitVersion version

6.0.0+Branch.main.Sha.d5d903370f6545aa6cd03c4d4729788df52efb5a

### Operating system

Linux, Windows

### What are you seeing?

When executing:
`dotnet-gitversion.exe /overrideconfig increment=Minor`
With an existing tag of 1.0.0, the version output is 1.0.1.

### What is expected?

My understanding of the documentation is that, if I do not place filtered words into my commit messages, I can set the type of version bump by executing gitversion like so:
`dotnet-gitversion.exe /overrideconfig increment=Patch`
In this example, the version increment type would become patch, and if my version was 1.0.0, it would become 1.0.1.
Similarly,
`dotnet-gitversion.exe /overrideconfig increment=Minor`
...would produce 1.1.0.

Unfortunately, we cannot use the automatic methods of determining version bump type - it must be specified by the build pipeline that is executed.

### Steps to Reproduce

- Create tag on first commit: 1.0.0
- Create 1+ commits.
- Run dotnet-gitversion.exe /overrideconfig increment=Minor

**Comment by dbieber-rfideas** [comment]
Additional notes:
- I do not have a GitVersion.yml file in the repository. I am using it with defaults.
- My main branch is called "main".

**Comment by arturcic** [comment]
please use version 6.0.0 instead of 5.12.0

**Comment by dbieber-rfideas** [comment]
I've updated to 6.0.0+Branch.main.Sha.d5d903370f6545aa6cd03c4d4729788df52efb5a
The behavior is the same.
I just need a way to force the type of version bump - is this the correct way to do it?

**Comment by dbieber-rfideas** [comment]
I've looked into it some more.

1. Using version 6 is problematic since the documentation is outdated for that version (is there a place I can see documentation for version 6?).
2. Adding a default config file and changing to "increment: Inherit" for all branches and the defaults gives the behavior I'm looking for.
3. Version 6 is appending a pre-release tag, even if the version comes from 'main'. I can't figure out how to remove the pre-release tag.

I tried to revert to version 5.x again, starting from a default config, and setting increment to Inherit there. That does not have any effect - version incrementing is completely ignored.

So, I'm not sure what to do now. Is there a way to disable the pre-release tag in the SemVer for version 6.x?
